### PR TITLE
SFI:ROPC metadata

### LINKED ISSUE #33727: Add meta-data showing Rope-See is completed
### Description

[Enter feedback here]

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/security/authorization/secure-data?view=aspnetcore-8.0

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/security/authorization/secure-data.md

### Document ID

7c151ed5-fd78-84fd-a08a-88912990c4ca

### Article author

@Rick-Anderson

[Related Issues](https://github.com/dotnet/AspNetCore.Docs/issues?q=is%3Aissue+is%3Aopen+7c151ed5-fd78-84fd-a08a-88912990c4ca)

### PR DESCRIPTION
**Pull request by Rick-Anderson**
I think we should standardize on ROPC hits that have been fixed.

Fixes  #33727

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/security/authorization/secure-data.md](https://github.com/dotnet/AspNetCore.Docs/blob/fee61532efcb8ed614d415e222e4b75fc8ea82ff/aspnetcore/security/authorization/secure-data.md) | [Create an ASP.NET Core web app with user data protected by authorization](https://review.learn.microsoft.com/en-us/aspnet/core/security/authorization/secure-data?branch=pr-en-us-33726) |

### CODE REVIEW

**Review by wadepickett — APPROVED** [review]
Looks good.  "Non-weak" seems a little awkward, but keeping the goal in mind, I didn't come up with anything better without being wordy.

File: aspnetcore/security/authorization/secure-data.md [hunk 1]
```diff
@@ -293,7 +294,7 @@ In the preceding code:
 
 If you haven't already set a password for seeded user accounts, use the [Secret Manager tool](xref:security/app-secrets#secret-manager) to set a password:
 
-* Choose a strong password: Use eight or more characters and at least one upper-case character, number, and symbol.
+* Choose a non-weak password: Use eight or more characters and at least one upper-case character, number, and symbol. Note those requirements don't guarantee a strong password.
```
Comment (tdykstra) [inline aspnetcore/security/authorization/secure-data.md:297]:
```suggestion
* Choose a [strong password](https://support.microsoft.com/en-us/windows/create-and-use-strong-passwords-c5cebb49-8c53-4f5e-2bc4-fe357ca048eb):
  * At least 12 characters long but 14 or more is better.
  * A combination of uppercase letters, lowercase letters, numbers, and symbols.
  * Not a word that can be found in a dictionary or the name of a person, character, product, or organization.
  * Significantly different from your previous passwords.
 * Easy for you to remember but difficult for others to guess. Consider using a memorable phrase like "6MonkeysRLooking^".
```

**Review by tdykstra — APPROVED** [review]
Suggestion could be made into an include.

File: aspnetcore/security/authorization/secure-data.md [hunk 2]
```diff
@@ -293,7 +294,12 @@ In the preceding code:
 
 If you haven't already set a password for seeded user accounts, use the [Secret Manager tool](xref:security/app-secrets#secret-manager) to set a password:
 
-* Choose a strong password: Use eight or more characters and at least one upper-case character, number, and symbol.
+* Choose a [strong password](https://support.microsoft.com/en-us/windows/create-and-use-strong-passwords-c5cebb49-8c53-4f5e-2bc4-fe357ca048eb):
```
Comment (Rick-Anderson) [inline aspnetcore/security/authorization/secure-data.md:297]:
```suggestion
* Choose a [strong password](https://support.microsoft.com/en-us/windows/create-and-use-strong-passwords-c5cebb49-8c53-4f5e-2bc4-fe357ca048eb). Although the following password isn't strong, it meets the minimum requirements:
```

**Comment by Rick-Anderson** [comment]
@tdykstra see 7dc93a3279e8517baa89dfb414533d61bf8791df

File: aspnetcore/security/authorization/secure-data.md [hunk 3]
```diff
@@ -293,7 +294,12 @@ In the preceding code:
 
 If you haven't already set a password for seeded user accounts, use the [Secret Manager tool](xref:security/app-secrets#secret-manager) to set a password:
 
-* Choose a strong password: Use eight or more characters and at least one upper-case character, number, and symbol.
+* Choose a [strong password](https://support.microsoft.com/en-us/windows/create-and-use-strong-passwords-c5cebb49-8c53-4f5e-2bc4-fe357ca048eb). Although the following password isn't strong, it meets the minimum requirements:
```
Comment (Rick-Anderson) [inline aspnetcore/security/authorization/secure-data.md:297]:
```suggestion
* Choose a [strong password](https://support.microsoft.com/en-us/windows/create-and-use-strong-passwords-c5cebb49-8c53-4f5e-2bc4-fe357ca048eb):
```

**Comment by Rick-Anderson** [comment]
> ~@tdykstra see [7dc93a3](https://github.com/dotnet/AspNetCore.Docs/commit/7dc93a3279e8517baa89dfb414533d61bf8791df)~

Never mind, it used to list the minimum requirements which wasn't strong, so I reverted.

**Pull request merged**